nmostesting/NMOSTesting.py

```diff
@@ -739,6 +762,42 @@ class ExitCodes(IntEnum):
     FAIL = 2  # Worst case test was a failure in non-interactive mode
 
 
+@core_app.route('/api', methods=["GET", "POST"])
+def api():
+    if not request.is_json:
+        return "Error: Request mimetype is not set to a JSON specific type with a valid JSON Body", 400

```

Comment:
Will this return JSON or text? If the latter could we make it JSON to keep the formats consistent?